File: src/backend/VoteBem/Entities/ResultadoTurno.cs
namespace VoteBem.Entities
{
    public class ResultadoTurno
    {
        public long SqCandidato { get; private set; }
        public long CdEleicao { get; private set; }
        public int NrTurno { get; private set; }
        public int NrVotos { get; private set; }
        public int? CdSitTotTurno { get; private set; }
        public string? DsSitTotTurno { get; private set; }

        public Candidatura Candidatura { get; private set; } = null!;
        public Eleicao Eleicao { get; private set; } = null!;

        protected ResultadoTurno() { }
    }
}

File: etl/models/resultado_turno.py
from sqlalchemy import Column, BigInteger, Integer, String, ForeignKey, ForeignKeyConstraint, PrimaryKeyConstraint
from .base import Base

class ResultadoTurno(Base):
    __tablename__ = "resultado_turno"

    sq_candidato = Column(BigInteger, ForeignKey("candidatura.sq_candidato"), nullable=False)
    cd_eleicao = Column(BigInteger, nullable=False)
    nr_turno = Column(Integer, nullable=False)
    nr_votos = Column(Integer, default=0)
    cd_sit_tot_turno = Column(Integer)
    ds_sit_tot_turno = Column(String)

    __table_args__ = (
        PrimaryKeyConstraint("sq_candidato", "nr_turno"),
        ForeignKeyConstraint(
            ["cd_eleicao", "nr_turno"],
            ["eleicao.cd_eleicao", "eleicao.nr_turno"]
        ),
    )
